build_msvc/libbitcoin_common/libbitcoin_common.vcxproj.in

```diff
@@ -8,6 +8,7 @@
     <ConfigurationType>StaticLibrary</ConfigurationType>
   </PropertyGroup>
   <ItemGroup>
+    <ClCompile Include="..\..\src\common\url.cpp" />
```

Comment:
@sipsorcery @hebasto is this correct?

Reply:
> @sipsorcery @hebasto is this correct?

Yes, it is. See https://github.com/bitcoin/bitcoin/pull/26294#discussion_r1009203613